rules/0520-vulnerability-detector_rules.xml

```diff
@@ -46,7 +46,7 @@
       <if_sid>23501</if_sid>
       <options>no_full_log</options>
       <field name="vulnerability.severity">High</field>
-      <description>$(vulnerability.title)</description>
+      <description>$(vulnerability.cve) affects $(vulnerability.package.name).</description>
```

Comment:
Remove the dot at the end of the description

Reply:
DONE